components/sections/SponsorShowcase.tsx

```diff
@@ -30,8 +31,8 @@ const sponsorsThinGrid = toGridTemplate([
   `adga adga . . ciena ciena`,
   `. . morgan morgan . .`,
   `. . balsamiq balsamiq . .`,
-  `voiceflow voiceflow . . onepassword onepassword`,
-  `. . wolfram wolfram . .`,
+  `voiceflow voiceflow wolfram wolfram onepassword onepassword`,
+  `. . thecodex thecodex . .`,
```

Comment:
Nitpick: inconsistent in-kind sponsor ordering between desktop and mobile grid (1Password and Wolfram switch)

Desktop:
![image](https://user-images.githubusercontent.com/20251243/105432543-cdba0e00-5c25-11eb-889b-9d9adde6bbc6.png)

Mobile:
![image](https://user-images.githubusercontent.com/20251243/105432580-de6a8400-5c25-11eb-87bb-4b4f4592da04.png)